api-watch.opam.template

```diff
@@ -0,0 +1,34 @@
+opam-version: "2.0"
```

Comment:
The template file only needs to define the `pin-depends` field. The other fields will be generated by dune.

The way opam.template files work with dune is that if it finds a field defined in the template it will use the field as it is in the template and will generate it otherwise. Here it basically overrides the whole generation process.